discovery/controllers/decorators/addon.go

```diff
@@ -151,7 +154,8 @@ func (a *Addon) AddComponents(components ...runtime.Object) error {
 		return err
 	}
 
-	var refs []discoveryv1alpha1.RichReference
+	refs := make([]discoveryv1alpha1.RichReference, 0)
+	metas := make([]discoveryv1alpha1.Metadata, 0)
```

Comment:
Is there a reason to use `make? we lose the "useful zero" value when we do this.